.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -0,0 +1,35 @@
+# SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+# SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+#
+# SPDX-License-Identifier: MIT OR Apache-2.0
+
+name: cxx-qt tests
+on: [push, pull_request]
+jobs:
+  ctest:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      # Install Rust toolchain
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
```

Comment:
You should be able to use `components: rustfmt, clippy` with the `actions-rs/toolchain@v1` and omit the next two `run:` commands.

Reply:
Done :-)